framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -2327,4 +2232,37 @@ public AnnotatedTypeMirror getDummyAssignedTo(ExpressionTree expressionTree) {
         }
         return null;
     }
+
+    /**
+     * Evaluates a given annotation to its primitive integral TypeKind equivalent.
+     *
+     * @param type {@link javax.lang.model.type.TypeKind}
+     * @return one of INT, SHORT, BYTE, CHAR, LONG
+     */
+    private static final TypeKind toPrimitiveIntegralTypeKind(TypeMirror type) {
+        TypeKind typeKind = NumberUtils.unboxPrimitive(type);
+        if (isPrimitiveIntegral(typeKind)) {
+            return typeKind;
+        }
+        throw new BugInCF(type.toString() + " expected to be an integral type.");
+    }
+
+    /**
+     * Return true if the argument is one of INT, SHORT, BYTE, CHAR, LONG.
```

Comment:
How about `Determine whether the argument is one of INT, SHORT, BYTE, CHAR, LONG or not.`
Or use `if and only if`.

Reply:
I opt for tacit over verbose. 
When dealing with private static functions of minimal logic, it is more burdensome than helpful to expand in prose in the javadoc.  There is only one argument and the return is boolean.

Reply:
I prefer to avoid the phrase "determine whether" because it isn't explicit about what the routine returns (it might be the negation of the determined value), so it makes me read the documentation twice to understand it.

I think "if and only if" is fine, and so is the current version is acceptable because "if and only if" is understood.  To avoid verbosity that can make the documentation harder to read, I sometimes use "iff" which is a standard mathematical abbreviation for "if and only if", but not everyone knows that abbreviation.